core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFolderReference.java

```diff
@@ -34,16 +35,15 @@ public ExchangeFolderReference(String folderName, String staticMailAddress, Fold
 		if (items.length > 1) {
 			this.mailbox = items[0];
 			this.folderName = items[1];
-			this.baseFolderId = null;
 		} else {
 			if(staticMailAddress == null){
 				throw new IllegalArgumentException("Cannot create ExchangeFolderReference when staticMailAddress is null " +
 					"and folderName does not contain separator ["+separator+"]!");
 			}
 			this.mailbox = staticMailAddress;
 			this.folderName = folderName;
-			this.baseFolderId = defaultBaseFolderId;
 		}
+		this.baseFolderId = defaultBaseFolderId;
```

Comment:
Dit klopt niet, ook al klopt de oorspronkelijke code ook niet.
De defaultBaseFolderId is alleen geldig als de mailbox gelijk is aan de defaultMailbox.

Reply:
Ik snap wat je bedoeld..